src/thor/isochrone_action.cc

```diff
@@ -13,12 +13,18 @@ std::string thor_worker_t::isochrones(Api& request) {
   auto costing = parse_costing(request);
   auto& options = *request.mutable_options();
 
-  std::vector<float> contours;
-  std::unordered_map<float, std::string> colors;
+  std::vector<float> contours_time, contours_distance;
```

Comment:
so the way it works now:

- each PBF contour **always** has time and distance set (can of course be 0)
- collect them into separate vectors and sort descending
- only generate contours if the highest metric is greater than 0
- skip the geometry in the serialization if it's a 0 contour